libcxx/include/__configuration/platform.h

```diff
@@ -42,11 +42,15 @@
 #  endif
 #endif
 
-// This is required in order for _NEWLIB_VERSION to be defined in places where we use it.
-// TODO: We shouldn't be including arbitrarily-named headers from libc++ since this can break valid
-//       user code. Move code paths that need _NEWLIB_VERSION to another customization mechanism.
-#if __has_include(<picolibc.h>)
-#  include <picolibc.h>
+// TODO: Remove this deprecated behavior after LLVM 22 release
```

Comment:
Since this is something that vendors should change, I'm not sure there's a benefit from deprecating this before we remove it.

Reply:
Agreed, I do not expect many people using it anyway - removed.